pkgs/development/python-modules/hopcroftkarp/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "hopcroftkarp";
+  version = "1.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "cc6fc7ad348bbe5c9451f8116845c46ae26290c92b2dd14690aae2d55ba5e3a6";
+  };
+
+  # tests fail due to bad package name
```

Comment:
The package has not been updated in over 3 years but is a necessary dependency of `matchpy`. The tests import `HopcroftKarp` instead of `hopcroftkarp` which is the package name. I could `substituteInPlace ....` but I didn't think that it was worth it. https://github.com/sofiatolaosebikan/hopcroftkarp/blob/master/test/test_hopcroftkarp.py#L18